app/src/main/java/ch/epfl/favo/common/DatabaseWrapper.java

```diff
@@ -2,58 +2,60 @@
 
 import android.util.Log;
 
-import com.google.firebase.firestore.CollectionReference;
 import com.google.firebase.firestore.FirebaseFirestore;
 import com.google.firebase.firestore.FirebaseFirestoreSettings;
 
 import java.util.Map;
-import java.util.Random;
 
 public class DatabaseWrapper {
 
-    private static final String TAG = "DatabaseWrapper";
-    private static DatabaseWrapper INSTANCE = null;
-    private FirebaseFirestore firestore;
-
-    // final fields regarding ID generation
-    private static String ID_CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
-    private static final int ID_LENGTH = 25;
-
-    private DatabaseWrapper() {
-        FirebaseFirestore.setLoggingEnabled(true);
-        FirebaseFirestoreSettings settings =
-                new FirebaseFirestoreSettings.Builder().setPersistenceEnabled(true).build();
-        try {
-            firestore = FirebaseFirestore.getInstance();
-            firestore.setFirestoreSettings(settings);
-        } catch (Exception e) {
-            Log.d(TAG, e.toString());
-        }
+  private static final String TAG = "DatabaseWrapper";
+  private static final int ID_LENGTH = 25;
+  private static DatabaseWrapper INSTANCE = null;
+  // final fields regarding ID generation
+  private static String ID_CHARS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789";
+  private FirebaseFirestore firestore;
```

Comment:
This comment is now in the wrong place since `ID_LENGTH` is now above the comment.

Reply:
yeah the fields were correctly sorted before, can we please move them back into that order

Reply:
Changed as it was before